chimedb/dataflag/orm.py

```diff
@@ -261,6 +261,12 @@ class DataFlagOpinion(DataSubset):
     client = pw.ForeignKeyField(DataFlagClient)
     revision = pw.ForeignKeyField(DataRevision, backref="opinions")
 
+    class Meta:
+        indexes = (
+            # create a unique index
+            (("type", "user", "start_time", "finish_time"), True),
```

Comment:
What will the conflict resolution strategy be?

Reply:
in bondia the user is warned that they already inserted an opinion, if they click anyways, the `decision` field is updated.